### PR TITLE
fix(tests): add test matrix for mysql and memory

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

**Comment by vladikoff** [comment]
@rfk r?

Similar to https://github.com/mozilla/fxa-auth-server/blob/master/.travis.yml#L50

File: .travis.yml [hunk 1]
```diff
@@ -22,14 +22,23 @@ notifications:
     skip_join: false
 
 before_install:
+  - if [ "$TRAVIS_NODE_VERSION" = "0.10" ]; then npm install -g npm@2; fi
```
Comment (pdehaan) [inline .travis.yml]: Should this be `==`, like on L37 and L41?

**Comment by vladikoff** [comment]
Ryan might be away for a few days, @vbudhram @jrgm r?

**Comment by jrgm** [comment]
r+ LGTM

**Pull request merged**